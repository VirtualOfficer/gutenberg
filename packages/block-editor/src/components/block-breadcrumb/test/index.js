/**
 * External dependencies
 */
import { shallow } from 'enzyme';

/**
 * Internal dependencies
 */
import BlockBreadcrumb from '../';

describe( 'BlockBreadcrumb', () => {
	test( 'should match snapshot', () => {
		const wrapper = shallow( <BlockBreadcrumb /> );
		expect( wrapper ).toMatchSnapshot();
	} );

	test( 'should match snapshot with root label', () => {
		const wrapper = shallow( <BlockBreadcrumb rootLabelText="Tuhinga" /> );
		expect( wrapper ).toMatchSnapshot();
	} );
} );
