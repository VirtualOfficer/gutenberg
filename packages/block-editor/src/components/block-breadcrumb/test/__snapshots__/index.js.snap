// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockBreadcrumb should match snapshot 1`] = `
<ul
  aria-label="Block breadcrumb"
  className="block-editor-block-breadcrumb"
  role="list"
>
  <li
    aria-current="true"
    className="block-editor-block-breadcrumb__current"
  >
    Document
  </li>
</ul>
`;

exports[`BlockBreadcrumb should match snapshot with root label 1`] = `
<ul
  aria-label="Block breadcrumb"
  className="block-editor-block-breadcrumb"
  role="list"
>
  <li
    aria-current="true"
    className="block-editor-block-breadcrumb__current"
  >
    Tuhinga
  </li>
</ul>
`;
